recipes/pyart_mch/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "pyart_mch" %}
+{% set version = "0.4.1" %}
+{% set sha256 = "6e16d4b947467b205f32aac9f8e309749d5f355babdae2a2467951df74e452ea" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script:  # [not win]
```

Comment:
if you want to skip windows you can do it using the ``skip`` as well

Reply:
as you are skipping windows now, you can remove the selectors ``# [not win]`` from the recipe :)